### PR TITLE
Update kernel to 5.4.72, Address 54 kernel CVEs, Add license file

### PR DESCRIPTION
**Pull request by christopherco**
###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 

Update kernel source to [5.4.72](https://github.com/microsoft/WSL2-Linux-Kernel/tree/linux-msft-5.4.72). New kernel source contains fixes for many kernel CVEs flagged by our tooling so address the CVEs. As part of this update, also add the kernel COPYING file to the packages missing the license file.

###### Change Log  

- Update kernel-headers, kernel, kernel-hyperv, and hyperv-daemons specs to use 5.4.72
- Add kernel COPYING license file to kernel-headers.spec, kernel.spec, kernel-hyperv.spec
- Update toolchain to use 5.4.72 source when building kernel headers
- Mark the following CVEs as fixed: CVE-2018-1000026, CVE-2018-16880, CVE-2020-12464, CVE-2020-12465, CVE-2020-12659, CVE-2020-15780, CVE-2020-14356, CVE-2020-14386, CVE-2020-25645, CVE-2020-25643, CVE-2020-25211, CVE-2020-25212, CVE-2008-4609, CVE-2020-14331, CVE-2010-0298, CVE-2020-10690, CVE-2020-25285, CVE-2020-10711, CVE-2019-3887, CVE-2020-14390, CVE-2019-19338, CVE-2019-20810, CVE-2020-10766, CVE-2020-10767, CVE-2020-10768, CVE-2020-10781, CVE-2020-12768, CVE-2020-14314, CVE-2020-14385, CVE-2020-25641, CVE-2020-26088, CVE-2020-10942, CVE-2020-12826, CVE-2019-3016, CVE-2019-3819, CVE-2020-16166, CVE-2020-11608, CVE-2020-11609, CVE-2020-25284, CVE-2020-12888, CVE-2017-8244, CVE-2017-8245, CVE-2017-8246, CVE-2009-4484, CVE-2015-5738, CVE-2007-4998, CVE-2010-0309, CVE-2011-0640, CVE-2020-12656, CVE-2011-2519, CVE-1999-0656, CVE-2010-4563, CVE-2019-20794, CVE-1999-0524
- Update cgmanifest's download URLs to point to 5.4.72 source location

###### Does this affect the toolchain?  

**YES**

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-2018-1000026
- https://nvd.nist.gov/vuln/detail/CVE-2018-16880
- https://nvd.nist.gov/vuln/detail/CVE-2020-12464
- https://nvd.nist.gov/vuln/detail/CVE-2020-12465
- https://nvd.nist.gov/vuln/detail/CVE-2020-12659
- https://nvd.nist.gov/vuln/detail/CVE-2020-15780
- https://nvd.nist.gov/vuln/detail/CVE-2020-14356
- https://nvd.nist.gov/vuln/detail/CVE-2020-14386
- https://nvd.nist.gov/vuln/detail/CVE-2020-25645
- https://nvd.nist.gov/vuln/detail/CVE-2020-25643
- https://nvd.nist.gov/vuln/detail/CVE-2020-25211
- https://nvd.nist.gov/vuln/detail/CVE-2020-25212
- https://nvd.nist.gov/vuln/detail/CVE-2008-4609
- https://nvd.nist.gov/vuln/detail/CVE-2020-14331
- https://nvd.nist.gov/vuln/detail/CVE-2010-0298
- https://nvd.nist.gov/vuln/detail/CVE-2020-10690
- https://nvd.nist.gov/vuln/detail/CVE-2020-25285
- https://nvd.nist.gov/vuln/detail/CVE-2020-10711
- https://nvd.nist.gov/vuln/detail/CVE-2019-3887
- https://nvd.nist.gov/vuln/detail/CVE-2020-14390
- https://nvd.nist.gov/vuln/detail/CVE-2019-19338
- https://nvd.nist.gov/vuln/detail/CVE-2019-20810
- https://nvd.nist.gov/vuln/detail/CVE-2020-10766
- https://nvd.nist.gov/vuln/detail/CVE-2020-10767
- https://nvd.nist.gov/vuln/detail/CVE-2020-10768
- https://nvd.nist.gov/vuln/detail/CVE-2020-10781
- https://nvd.nist.gov/vuln/detail/CVE-2020-12768
- https://nvd.nist.gov/vuln/detail/CVE-2020-14314
- https://nvd.nist.gov/vuln/detail/CVE-2020-14385
- https://nvd.nist.gov/vuln/detail/CVE-2020-25641
- https://nvd.nist.gov/vuln/detail/CVE-2020-26088
- https://nvd.nist.gov/vuln/detail/CVE-2020-10942
- https://nvd.nist.gov/vuln/detail/CVE-2020-12826
- https://nvd.nist.gov/vuln/detail/CVE-2019-3016
- https://nvd.nist.gov/vuln/detail/CVE-2019-3819
- https://nvd.nist.gov/vuln/detail/CVE-2020-16166
- https://nvd.nist.gov/vuln/detail/CVE-2020-11608
- https://nvd.nist.gov/vuln/detail/CVE-2020-11609
- https://nvd.nist.gov/vuln/detail/CVE-2020-25284
- https://nvd.nist.gov/vuln/detail/CVE-2020-12888
- https://nvd.nist.gov/vuln/detail/CVE-2017-8244
- https://nvd.nist.gov/vuln/detail/CVE-2017-8245
- https://nvd.nist.gov/vuln/detail/CVE-2017-8246
- https://nvd.nist.gov/vuln/detail/CVE-2009-4484
- https://nvd.nist.gov/vuln/detail/CVE-2015-5738
- https://nvd.nist.gov/vuln/detail/CVE-2007-4998
- https://nvd.nist.gov/vuln/detail/CVE-2010-0309
- https://nvd.nist.gov/vuln/detail/CVE-2011-0640
- https://nvd.nist.gov/vuln/detail/CVE-2020-12656
- https://nvd.nist.gov/vuln/detail/CVE-2011-2519
- https://nvd.nist.gov/vuln/detail/CVE-1999-0656
- https://nvd.nist.gov/vuln/detail/CVE-2010-4563
- https://nvd.nist.gov/vuln/detail/CVE-2019-20794
- https://nvd.nist.gov/vuln/detail/CVE-1999-0524
###### Test Methodology

- Local full toolchain, all package, and core image build. Tested vhdx image boots with the proper kernel version.

### CODE REVIEW

File: SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec [hunk 1]
```diff
@@ -51,7 +46,7 @@ mkdir -p %{buildroot}/%{_localstatedir}/lib/initramfs/kernel
 
 cp -rp ./boot/* %{buildroot}/boot
 cp -rp ./lib/* %{buildroot}/lib
-cp -rp ./var/* %{buildroot}/%{_localstatedir}
+cp -rp .%{_var}/* %{buildroot}/%{_localstatedir}
```
Comment (PawelWMS) [inline SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec:49]: I'd argue that this is an issue with the linter and not the spec. `%{_var}` is tied to how files are organized on the final system and here we're accessing the source's internal directory, so it's not the same thing/purpose. Not to mention, that if we ever change `${_var}` (doubtful) we will break this spec.

Btw. I believe linter's config is still being tweaked, so feel free to report any suggestions to @thcrain-msft.

**Review by PawelWMS — CHANGES_REQUESTED** [review]

File: SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec (hunk 1 above)
Comment (christopherco) [reply SPECS-SIGNED/kernel-signed-aarch64/kernel-signed-aarch64.spec]: @PawelWMS Thanks for pointing this out. Restored the ./var value. Will follow up on the linter side

**Review by PawelWMS — APPROVED** [review]
:shipit:

File: SPECS/kernel/kernel.signatures.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
  "Signatures": {
-  "config": "b8c9e2a875e4e6655fdbeb626088529fd1cef401b8f67a481fc301d2a3a026c5",
+  "config": "681e8ff274fc3fc79eaf38d5152e99702e22346bd2aaff06925715dcf1a00fc3",
   "config_aarch64": "3057cf5c5f04b57c4d69f9783d4809de217fb46a4278694c19f6c3ffd81249c5",
```
Comment (dmcilvaney) [inline SPECS/kernel/kernel.signatures.json:4]: Are there no changes to the aarch64 config?

Comment (christopherco) [reply SPECS/kernel/kernel.signatures.json:4]: Good catch! Checking to see what happened here...

Comment (christopherco) [reply SPECS/kernel/kernel.signatures.json]: Fixed

**Comment by christopherco** [comment]
At this stage, linter is only flagging `%{__cc}` vs `gcc`.
```
-%{__cc} %{optflags} -c hv_kvp_daemon.c
+gcc %{optflags} -c hv_kvp_daemon.c
```
Discussed with @thcrain-msft - In this case, linter should allow `%{__cc}`. Currently the `%{__cc}` macro will be `gcc` by default but this may not always be the case.

**Review by dmcilvaney — APPROVED** [review]

**Pull request merged**